src/packages/src/xroad/redhat/SPECS/xroad-base.spec

```
@@ -111,18 +111,20 @@ exit 1
 fi
 
 %post
-umask 007
+umask 027
 
 # ensure home directory ownership
 mkdir -p /var/lib/xroad/backup
 su - xroad -c "test -O /var/lib/xroad && test -G /var/lib/xroad" || chown xroad:xroad /var/lib/xroad
 chown xroad:xroad /var/lib/xroad/backup
-chmod 0775 /var/lib/xroad
+chmod 0755 /var/lib/xroad
+chmod -R go-w /var/lib/xroad
 
 # nicer log directory permissions
-mkdir -p -m1770 /var/log/xroad
-chmod 1770 /var/log/xroad
+mkdir -p -m1750 /var/log/xroad
+chmod 1750 /var/log/xroad
 chown xroad:adm /var/log/xroad
+chmod -R 1750 /var/log/xroad
```

This will set executable and "sticky" bits also for all regular files. 
```suggestion
chmod -R go-w /var/log/xroad
```